templates/starter/bicep/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure Developer CLI (azd) Bicep Starter
+
+A starter template with [Bicep](https://aka.ms/bicep) as infrastructure provider for [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview) (azd).
```

Comment:
@gkulin Can you take a look at the READMEs to make sure they fit with the current updates you are proposing for template README updates?

Mainly, do these clearly explain to developers what these repos are and how to use them?

Reply:
If we want the short description to follow the same formatting from #2073, maybe we could try something like:

> A jumpstart for getting your app on Azure. Add your own application code and leverage the Infrastructure as Code assets (written in Bicep) to get up and running quickly. 

I see `azure.yaml` currently specifies Python, Node.js, and app service. But I think we probably don't need to mention those in the readmes since there isn't anything else in the starter explicitly tied to those tools

Reply:
Worked with grace to come up with the final text here. Let's follow-up separately for any minor suggestions.